tmt/steps/report/display.py

```diff
@@ -26,11 +26,14 @@ def details(self, result: tmt.Result, verbosity: int) -> None:
         # -vv and more follows
         # TODO: are we sure it cannot be None?
         assert self.step.plan.execute.workdir is not None
-        for log_file in result.log:
-            log_name = os.path.basename(log_file)
-            full_path = os.path.join(self.step.plan.execute.workdir, log_file)
+        for _log_file in result.log:
+            # TODO: this should be done already, resul.log should use Path instead
```

Comment:
```suggestion
            # TODO: this should be done already, result.log should use Path instead
```

Reply:
Fixed in 3a5a9bacf6ceedddfec98aecab1720f57e28890b